tasks/src/feature-extraction/about.md

```diff
@@ -1,19 +1,21 @@
 # Use cases
 
+You can contribute this area with common use cases of the task!
+
 ## Sentence classification
 
-For sentence classification, we are interested in BERT’s hidden layer output (embeddings), so we get the sentence embedding using
-`feature-extraction` pipeline which contain contextual knowledge of English language that can be used as input for Tree based classifiers like Random Forest, XGBoost, CatBoost. 
-(see https://towardsdatascience.com/feature-extraction-with-bert-for-text-classification-533dde44dc2f).
+For sentence classification, we are interested in BERT’s hidden layer output (embeddings), so we get the sentence embeddings using
```

Comment:
Can you start this paragraph with some information on how feature extraction pipeline can be used and relates to text classification? Try to put yourself in a beginner's shoes 🙂

Reply:
Also if it's okay I feel like this section (sentence classification use case) is a bit intermediate level info compressed here, it's very nicely done and very relevant. Is it okay if we make this use case a nicely done colab instead and put it to useful resources section? I will share it separately too!